src/Framework/BuildCheck/BuildCheckEventArgs.cs

```diff
@@ -46,12 +46,7 @@ internal override void WriteToStream(BinaryWriter writer)
     {
         base.WriteToStream(writer);
 
-        writer.Write7BitEncodedInt(TracingData.InfrastructureTracingData.Count);
-        foreach (KeyValuePair<string, TimeSpan> kvp in TracingData.InfrastructureTracingData)
-        {
-            writer.Write(kvp.Key);
-            writer.Write(kvp.Value.Ticks);
-        }
+        writer.WriteDurationsDictionary(TracingData.InfrastructureTracingData);
```

Comment:
Is this just a refactor that slipped to this PR or is there a functional change I'm not seeing?

Reply:
Refactoring. I thought I'd need it, but didn't end up without it.
I can remove it